starfish/util/exec.py

```diff
@@ -0,0 +1,66 @@
+import json
+import os
+import shutil
+import subprocess
+import sys
+import tempfile
+import unittest
+from typing import Sequence
+
+import jsonpath_rw
+import numpy as np
+import pandas as pd
+
+from starfish.intensity_table import IntensityTable
+from starfish.types import Features
+from starfish.util import clock
+
+
+def stages(commands, subdirs=None, keep_data=False):
```

Comment:
Please add type-hinting, numpy-style docstring parameter explanations. (see codebase for examples, most should have this)